web/app/js/components/Navigation.jsx

```diff
@@ -235,7 +230,6 @@ class NavigationBase extends React.Component {
             { this.menuItem("/routes", "Top Routes", <Icon className={classNames("fas fa-random", classes.shrinkIcon)} />) }
             { this.menuItem("/servicemesh", "Service Mesh", <CloudQueueIcon className={classes.shrinkIcon} />) }
             <NavigationResources />
-            { this.menuItem("/debug", "Debug", <BuildIcon className={classes.shrinkIcon} />) }
```

Comment:
hey! I think you'll need to merge master into this branch. this shouldn't be removed.

Reply:
Good catch! Will do!

Reply:
@rmars should be resolved now.